lib/winston.js

```diff
@@ -135,6 +135,17 @@ Object.defineProperty(exports, 'exceptions', {
   }
 });
 
+/**
+ * Define getter for `rejections` which replaces `handleRejections` and
+ * `unhandleRejections`.
+ * @type {Object}
+ */
+Object.defineProperty(exports, 'rejections', {
```

Comment:
This doesn't get used anywhere atm but for completeness it felt like we should have this to parallel `exceptions` right above?